runtime/lua/vim/lsp.lua

```diff
@@ -1172,6 +1174,16 @@ function lsp.start_client(config)
     return nil, lsp.rpc_response_error(protocol.ErrorCodes.MethodNotFound)
   end
 
+  ---Logs the given error to the LSP log and to the error buffer.
+  ---@param code integer Error code
+  ---@param err any Error arguments
```

Comment:
https://github.com/neovim/neovim/pull/25345 was merged so this should now, officially, be `--- `

Reply:
Thanks for pointing this out :)